java-collections/src/main/java/dev/pawel/MyList.java

```diff
@@ -1,13 +1,31 @@
 package dev.pawel;
 
 public interface MyList<E> {
+
     E get(int index);
+
     E set(int index, E element);
+
     boolean add(E element);
+
     void add(int index, E element);
+
     E remove(int index);
+
     boolean remove(E element);
+
     int size();
+
     boolean contains(E element);
+
     int indexOf(E element);
+
+    boolean addAll(MyList<E> myList);
+
+    static <E> MyList<E> of(E... elements) {
+        MyList<E> myList = new MyArrayList<>();
```

Comment:
You can pass the capacity to the constructor of the new MyArrayList to avoid additional resizing